llvm/test/CodeGen/CHERI-Generic/RISCV32/atomic-rmw-cap-ptr.ll

```diff
@@ -30,6 +30,7 @@ define dso_local void @atomic_cap_ptr_xchg(i32 addrspace(200)* %ptr, i32 %val) n
 ;
 ; HYBRID-LABEL: atomic_cap_ptr_xchg:
 ; HYBRID:       # %bb.0: # %bb
+; HYBRID-NEXT:    fence rw, rw
```

Comment:
This file needs tests for orders other then seq_cst.

Reply:
Will do after updating for LR/SC